qri/dataset.py

```diff
@@ -125,7 +148,30 @@ def __repr__(self):
         return 'Dataset("%s")' % self.human_ref()
 
     def _repr_html_(self):
-        return '<code>Dataset("%s")</code>' % self.human_ref()
+        title = self.human_ref()
+        meta = self.meta_component
+        if meta is None:
```

Comment:
`meta` will never be `None`. As the code is written, the `meta_component` will be constructed as a `Meta` without any fields if none exists on the dataset. Instead, this should use local vars that are assigned the value from meta, or use a default otherwise.